test/ClangImporter/cxx_interop.swift

```diff
@@ -1,5 +1,5 @@
 // RUN: %empty-directory(%t)
-// RUN: %target-swift-frontend -typecheck %s -I %S/Inputs/custom-modules -module-cache-path %t -enable-cxx-interop
```

Comment:
It is, however, the `target-swift-frontend` (and `target-swiftxx-frontend`) substitutions already provide a module cache path.  Is there a reason to override the cache path here?  If so, I think we should explicitly call that out in the test.

Reply:
I'm pretty sure there's no reason to override the path. The `ClangImporter` tests are (relatively) old, and it may have been needed when they were written but they're not needed anymore.

Reply:
Yeah, I figured that it is better to do this in pieces.